src/features/projectsV2/ProjectDetails/components/ProjectDownloads.tsx

```diff
@@ -0,0 +1,47 @@
+import { Certificate, ProjectExpense } from '@planet-sdk/common';
+import styles from '../styles/ProjectInfo.module.scss';
+import { useTranslations } from 'next-intl';
+import SingleProjectInfoItem from './microComponents/SingleProjectInfoItem';
+import ProjectCertificates from './microComponents/ProjectCertificates';
+import ProjectExpenseReports from './microComponents/ProjectExpenseReports';
+
+interface Props {
+  certificates: Certificate[];
+  expenses: ProjectExpense[];
+}
+
+const ProjectDownloads = ({ certificates, expenses }: Props) => {
+  const tManageProjects = useTranslations('ManageProjects');
+
+  const financialAndCertificationData = [
+    {
```

Comment:
Let's rename `financelaAndCertificationData` --> `downloadsConfig`. 

It does not represent simple data, but a configuration of what should render and when, and we can also add another item (e.g. progressReports) without having to rename everything to make sense.